Source/ParameterEncoding.swift

```diff
@@ -99,15 +122,24 @@ public struct URLEncoding: ParameterEncoding {
     /// The destination defining where the encoded query string is to be applied to the URL request.
     public let destination: Destination
 
+    /// The convention for encoding array parameters.
+    public let arrayEncoding: ArrayEncoding
+
+    /// The convention for encoding boolean parameters.
+    public let boolEncoding: BoolEncoding
```

Comment:
As with `arrayEncoding`, it's a bit clearer to say "The encoding to use for `Bool` parameters."